specification/billing/resource-manager/Microsoft.Billing/stable/2023-04-01/invoiceSection.json

```diff
@@ -122,176 +197,322 @@
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.Billing/billingAccounts/{billingAccountName}/billingProfiles/{billingProfileName}/invoiceSections": {
+      "get": {
         "tags": [
-          "InvoiceSections"
+          "InvoiceSection"
+        ],
+        "description": "Lists the invoice sections that a user has access to. The operation is supported only for billing accounts with agreement type Microsoft Customer Agreement.",
+        "externalDocs": {
+          "url": "https://docs.microsoft.com/en-us/rest/api/billing/"
+        },
+        "operationId": "InvoiceSections_ListByBillingProfile",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "./types.json#/parameters/billingAccountName"
+          },
+          {
+            "$ref": "./types.json#/parameters/billingProfileName"
+          },
+          {
+            "in": "query",
+            "name": "includeDeleted",
+            "description": "Can be used to get deleted invoice sections.",
+            "type": "boolean",
+            "default": false
+          },
+          {
+            "$ref": "./types.json#/parameters/apiVersionParameter"
+          },
+          {
+            "$ref": "./types.json#/parameters/filter"
+          },
+          {
+            "$ref": "./types.json#/parameters/orderBy"
+          },
+          {
+            "$ref": "./types.json#/parameters/top"
+          },
+          {
+            "$ref": "./types.json#/parameters/skip"
+          },
+          {
+            "$ref": "./types.json#/parameters/count"
+          },
+          {
+            "$ref": "./types.json#/parameters/search"
+          }
         ],
+        "responses": {
+          "200": {
+            "description": "A list of invoice sections.",
+            "schema": {
+              "$ref": "#/definitions/InvoiceSectionListResult"
+            }
+          },
+          "default": {
+            "description": "Default response",
+            "schema": {
+              "$ref": "./types.json#/definitions/ArmErrorResponse"
+            }
+          }
+        },
         "x-ms-examples": {
-          "invoiceSectionsPut": {
-            "$ref": "./examples/invoiceSectionsPut.json"
+          "InvoiceSectionsListByBillingProfile": {
+            "$ref": "./examples/invoiceSectionsListByBillingProfile.json"
           }
         },
-        "x-ms-long-running-operation": true
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
       }
     }
   },
   "definitions": {
+    "DeleteInvoiceSectionEligibilityDetail": {
+      "description": "The details of delete invoice section eligibility result.",
+      "type": "object",
+      "properties": {
+        "code": {
+          "description": "Code for the delete invoice section validation.",
+          "enum": [
+            "Other",
+            "LastInvoiceSection",
+            "ActiveAzurePlans",
```

Comment:
We're trying to move away from using "Azure plans" as a concept. Also, I don't understand why the existence of an active Azure plan matters - why do we care as long as there are no billing subscriptions? Why can't we automatically delete the underlying rating asset as part of deleting the invoice section?

Reply:
This is the check we are making for `ActiveAzurePlans`

![image](https://github.com/Azure/azure-rest-api-specs/assets/29668917/c7935e57-a9dc-4c1b-85ac-81e095617703)

Reply:
We'll tackle this with the `enabledAzurePlans` item from above and will leave it as `ActiveAzurePlans`